internal/params/config.go

```diff
@@ -98,6 +99,7 @@ var (
 		SHA3Epoch:                  big.NewInt(74570),
 		HIP6And8Epoch:              big.NewInt(74570),
 		StakingPrecompileEpoch:     big.NewInt(75175),
+		ChainIdFixEpoch:            big.NewInt(75685),
```

Comment:
Please change this to `EpochTBD` too, and merge with branch `main`. Thank you.

Reply:
it's done, thanks